extensions/sql-migration/src/dashboard/sqlServerDashboard.ts

```diff
@@ -0,0 +1,510 @@
+			}
+		]);
+
+		const viewPanelStyle = {
+			'padding': '10px 5px 10px 10px',
+			'margin-top': '-15px'
+		};
+		linksContainer.addItem(videosContainer, {
+			CSSStyles: viewPanelStyle
+		});
+
+		return linksContainer;
+	}
+
+	private createLink(view: azdata.ModelView, linkMetaData: IActionMetadata): azdata.Component {
+		const maxWidth = 400;
+		const labelsContainer = view.modelBuilder.flexContainer().withLayout({
+			flexFlow: 'column',
+			width: maxWidth,
+			justifyContent: 'flex-start'
+		}).component();
+		const descriptionComponent = view.modelBuilder.text().withProperties({
+			value: linkMetaData.description,
+			width: maxWidth,
+			CSSStyles: {
+				'font-size': '12px',
+				'line-height': '16px',
+				'margin': '0px'
+			}
+		}).component();
+		const linkContainer = view.modelBuilder.flexContainer().withLayout({
+			flexFlow: 'row',
+			width: maxWidth + 10,
+			justifyContent: 'flex-start'
+		}).component();
+		const linkComponent = view.modelBuilder.hyperlink().withProps({
+			label: linkMetaData.title!,
+			url: linkMetaData.link!,
+			showLinkIcon: true,
+			CSSStyles: {
+				'font-size': '14px',
+				'margin': '0px'
+			}
+		}).component();
+		linkContainer.addItem(linkComponent, {
+			CSSStyles: {
+				'font-size': '14px',
+				'line-height': '18px',
+				'padding': '0 5px 0 0',
+			}
+		});
+		labelsContainer.addItems([linkContainer, descriptionComponent], {
+			CSSStyles: {
+				'padding': '5px 0 0 0',
+			}
+		});
+
+		return labelsContainer;
+	}
+
+	private createVideoLinkContainers(view: azdata.ModelView, links: IActionMetadata[]): azdata.Component {
+		const maxWidth = 400;
+		const videosContainer = view.modelBuilder.flexContainer().withLayout({
+			flexFlow: 'row',
+			width: maxWidth,
+		}).component();
+
+		links.forEach(link => {
+			const videoContainer = this.createVideoLink(view, link);
+
+			videosContainer.addItem(videoContainer);
+		});
+
+		return videosContainer;
+	}
+
+	private createVideoLink(view: azdata.ModelView, linkMetaData: IActionMetadata): azdata.Component {
+		const maxWidth = 150;
+		const videosContainer = view.modelBuilder.flexContainer().withLayout({
+			flexFlow: 'column',
+			width: maxWidth,
+			justifyContent: 'flex-start'
+		}).component();
+		const video1Container = view.modelBuilder.divContainer().withProperties({
+			clickable: true,
+			width: maxWidth,
+			height: '100px'
+		}).component();
+		const descriptionComponent = view.modelBuilder.text().withProperties({
+			value: linkMetaData.description,
+			width: maxWidth,
+			height: '50px',
+			CSSStyles: {
+				'font-size': '13px',
+				'margin': '0px'
+			}
+		}).component();
+		video1Container.onDidClick(async () => {
+			if (linkMetaData.link) {
+				await vscode.env.openExternal(vscode.Uri.parse(linkMetaData.link));
+			}
+		});
+		videosContainer.addItem(video1Container, {
+			CSSStyles: {
+				'background-image': `url(${vscode.Uri.file(this.asAbsolutePath(linkMetaData.iconPath?.light as string || ''))})`,
```

Comment:
Typically we have an IconHelper class to provide icons so that we don't have to re-implement the same logic in every place that needs an icon

Reply:
Yeah, I have already added that class. I will move it there.